roles/tm-deepsec-aws-setup/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+# tasks file for tm_silent_setup
+
+- name: Ensure bash, OpenSSl, and libssl are the latest versions
+  yum: name={{ item }} update_cache=true state=latest
+  with_items: "{{ pre_package }}"
+  tags: packages
+
+- name: Install psycopg2 python package for postgres
+  pip:
+    name: "{{ item }}"
+  with_items: "{{ pip_package }}"
```

Comment:
The same here, the `name` parameter of `pip` also accept list. Please remove the `with_items`.

Reply:
+1